internal/common/common.go

```diff
@@ -59,14 +59,22 @@ ipv6.dhcp-duid=ll
 `
 
 // Configuration to be used by MCO manifest to get consistent IPv6 DHCP client identification.
-const Ipv6DuidRuntimeConf = `
+const Ipv6DuidRuntimeConfPre410 = `
```

Comment:
I think that we should rename it Ipv6DuidRuntimeConfPre49 ( if i am not mistaken we don't need to set keyfile Path in 4.9 also

Reply:
You are correct, but on the other hand (just for the safety) I wanted to keep the current workflow in 4.9 because it works "as is". Indeed we have discovered that using keyfile path in 4.9 is not needed, but if we remove it with this patch, it needs that we need to do additional testing and potentially break stuff that already works (I know that we personally believe it will not break, but there may be always some corner case somewhere out there)

Reply:
ok, sure